harness/determined/ipc.py

```diff
@@ -32,9 +32,19 @@ def __init__(self, metrics: Dict[str, Any], num_batches: int):
 
 class ConnectedMessage:
```

Comment:
Was this class just dead code before?  It seems like you could use this new `ConnectedMessage` in place of the `ReadyMessage`, and then delete the `ConnectedMessageAck` and the `ReadyMessage` entirely, right?

Reply:
Yeah I had initially left it in because the `ReadyMessage` message served as a barrier before training starts, but have since realized that it is only per-machine barrier not a global one, so no point in keeping it.